components/dashboard/conf/Caddyfile

```diff
@@ -0,0 +1,44 @@
+{
+	auto_https off
+	admin off
+}
+
+(compression) {
+	encode zstd gzip
+}
+
+(caching) {
+	header Cache-Control "public, max-age=604800, must-revalidate"
```

Comment:
Until now caching was the responsibility of the "front-facing" `proxy` (cmp. [here](https://github.com/gitpod-io/gitpod/blob/main/chart/config/proxy/lib.locations.conf#L37-L50)).

This looks fine generally, but we should double check the results, and ideally either:
 a) cross-link with comments, or better
 b) merge into one place.